modules/apps/site-initializer/site-initializer-cms/src/main/resources/site-initializer/fragments/group/cms/fragments/page-bar/index.html

```diff
@@ -1,4 +1,4 @@
-<nav class="component-tbar page-bar tbar">
+<nav class="component-tbar navigation-bar-light page-bar position-fixed tbar w-100">
```

Comment:
`navigation-bar-light` shouldn't be mixed with `tbar`. These are two separate components and should be kept separate so styles don't get mixed up. You can use `bg-white` instead.

One thought I had was to use the deprecated `application-bar`. We can add a variant `application-bar-light` in Clay. It seems like a good fit since it was supposed to be used for the control-menu.

Reply:
We can also remove `position-fixed` and `w-100` if we use the sticky mentioned below.